iOS/ViewControllers/UpcomingMoviesViewController.swift

```diff
@@ -0,0 +1,98 @@
+import UIKit
+import Shared
+
+protocol ListUpcomingMoviesDelegate: class {
```

Comment:
I got confused here.
If this is a delegate, don't you think it should have the iOS pattern for delegate ?
```
func listUpcomingMovies(_ listUpcomingMovies: ListUpcomingMovies, didList movies: [Movie])
```

Now I don't know if this is a delegate or a protocol to implement. (behaviour)

It seems that this delegate should not be defined here.